build.gradle

```diff
@@ -31,13 +31,20 @@ dependencies {
     // logging
     implementation 'ch.qos.logback:logback-classic:1.2.3'
 
+    // json
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.12.3'
```

Comment:
The naming of things is definitely one of the more confusing things about Java for me. No way I would have guessed this library is for JSON. 😆

Reply:
Haha. Yeah. Jackson and GSON are the two popular JSON serializers in Java. I use GSON at work so naturally I wanted to try the other one here...